jenkins/docker/javascript.Dockerfile

```diff
@@ -39,4 +39,11 @@ RUN useradd --uid 1000 -ms /bin/bash ubuntu
 RUN mkdir -p /data/db \
 	&& chown -R ubuntu:ubuntu /data
 
+# install rust and wasm-pack
+ENV PATH=$PATH:/home/ubuntu/.cargo/bin
+RUN curl https://sh.rustup.rs -sSf | CARGO_HOME=/home/ubuntu/.cargo sh -s -- -y \
+	&& . "/home/ubuntu/.cargo/env" \
```

Comment:
not sure if there's a way to avoid source call
or if nicer way to install this as user (ubuntu) instead of root